addon/ng2/tasks/test.ts

```diff
@@ -33,10 +30,10 @@ module.exports = Task.extend({
 
       // Abstract the webpack concepts from the local karma config.
       // Add those details here.
-
+      
```

Comment:
Please undo. No whitespaces at the end of lines. Note to self; fix the linter to prevent this.